Gopkg.toml

```diff
@@ -45,6 +45,15 @@
   branch = "master"
   name = "github.com/evanphx/json-patch"
 
+# The main project has an open PR with a bugfix that affects us.
+# This jsonparser fork includes that PR.  Once merged, this can
+# return to the main project.
+# See: https://github.com/buger/jsonparser/pull/137
+[[constraint]]
+  branch = "master"
+  name = "github.com/buger/jsonparser"
+  source = "https://github.com/dbemiller/jsonparser.git"
```

Comment:
Do we want this fork to live in the prebid github org? Probably not worth it if we're intending to switch back to upstream after a single PR is merged.

Reply:
yeah... I considered that too. I chose not to put it in the prebid org just because I didn't want to pollute the namespace. Hopefully this is a short-lived fork.

We can always move it if it turns out to be a long-lived fork and we care that much. At this point, we don't even know for certain that this will get rid of the panics, or what other bugs it might cause... so that seemed a little premature.